src/backend/metal/src/command.rs

```diff
@@ -1092,6 +1102,13 @@ impl pool::RawCommandPool<Backend> for CommandPool {
     }
 }
 
+// Sets up the load/store operations. Returns `true` if the clear color needs to be set.
+fn set_operations(attachment: &metal::RenderPassAttachmentDescriptorRef, ops: AttachmentOps) -> bool {
+    attachment.set_load_action(conv::map_load_operation(ops.load));
+    attachment.set_store_action(conv::map_store_operation(ops.store));
+    ops.load == AttachmentLoadOp::Clear
```

Comment:
nit: seems a bit confusing to do this here

Reply:
yeah, I agree. It is sooo tempting though. Would it be cleaner if I return something like:
```rust
OperationFollowup {
  clear: bool,
}
```